src/static/js/modules/nav-primary.js

```diff
@@ -28,16 +37,26 @@ function init() {
     var $this = $( this );
     var $thisSubNav = $this.siblings( '.js-sub-nav' );
     var $otherSubNavs = $subNavs.not( $thisSubNav );
+    var $firstLink = $thisSubNav.find( 'a' )[0];
```

Comment:
Oh, it should probably be wrapped in a `$()` to make it a jQuery object, otherwise `.focus()` wouldn't work right?